### ISSUE TITLE
error message "KeyError: 'scale'" ; and could not find sample data on the galaxy

### ISSUE THREAD

**Issue by hsiaochiao** [deeptools/deepTools#1171]
questions: 1: error message "KeyError: 'scale'" 
I have installed the deeptools v3.5.1 on linux system using pip3 (phython3.7) and run the computeMatrix using the below code

computeMatrix scale-regions -S $path_file_bigWig/$bigWig_1 $path_file_bigWig/$bigWig_2 $path_file_bigWig/$bigWig_3 $path_file_bigWig/$bigWig_4 -R $path_file_bed -a 3000 -b 3000 -p 16 --skipZeros -o matrix1_H3K4me3.gz --outFileSortedRegions H3K4me3.bed

then I want to plot heatmap 

plotHeatmap  -m matrix1_H3K4me3.gz -out H3K4me3.png

but I got an error message "KeyError: 'scale'" . The full output message i got is below
------------------------------------------------------------------------------------------------------------------------
Traceback (most recent call last):
  File "/deepTools/v3.5.1/bin/plotHeatmap", line 5, in <module>
    from deeptools.plotHeatmap import main
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/deeptools/plotHeatmap.py", line 12, in <module>
    import matplotlib.pyplot as plt
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/pyplot.py", line 57, in <module>
    from matplotlib.figure import Figure, figaspect
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/figure.py", line 25, in <module>
    from matplotlib import _blocking_input, docstring, projections
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/projections/__init__.py", line 58, in <module>
    from mpl_toolkits.mplot3d import Axes3D
  File "/python/3.7.3/lib/python3.7/site-packages/matplotlib-3.1.3-py3.7-linux-x86_64.egg/mpl_toolkits/mplot3d/__init__.py", line 1, in <module>
    from .axes3d import Axes3D
  File "/python/3.7.3/lib/python3.7/site-packages/matplotlib-3.1.3-py3.7-linux-x86_64.egg/mpl_toolkits/mplot3d/axes3d.py", line 42, in <module>
    class Axes3D(Axes):
  File "/python/3.7.3/lib/python3.7/site-packages/matplotlib-3.1.3-py3.7-linux-x86_64.egg/mpl_toolkits/mplot3d/axes3d.py", line 52, in Axes3D
    azim=-60, elev=30, zscale=None, sharez=None, proj_type='persp',
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/docstring.py", line 84, in __call__
    super().__call__(obj)
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/docstring.py", line 38, in __call__
    func.__doc__ = inspect.cleandoc(func.__doc__) % self.params
  File "/deepTools/v3.5.1.new/lib/python3.7/site-packages/matplotlib/docstring.py", line 57, in __missing__
    raise KeyError(key)
KeyError: 'scale'

I don't know where is the problem.

------------------------------------------------------------------------------------------------------------------------

question 2 could not find sample data on the galaxy

because this is my first time using deeptools so I want to use the sample date to try the software.

I found the website indicates the sample data can be downloaded from galaxy.
https://deeptools.readthedocs.io/en/develop/content/example_step_by_step.html
"If you would like to try out the protocols with sample data, go to [deepTools Galaxy](http://deeptools.ie-freiburg.mpg.de/) –> “Shared Data” –> “Data Libraries” –> “deepTools Test Files”. Simply select BED/BAM/bigWig files and click, “to History”. You can also download the test data sets to your computer by clicking “Download” at the top."

But there is no sample data on the galaxy.

Dose anyone has the sample data I can download?

Thank you

**Comment by adRn-s** [comment]
Test data is available [here](https://github.com/deeptools/deepTools/tree/master/deeptools/test/test_data). Let us know if your commands keep failing even with these files.

Only after being able to use a given command on test data, you should try it on your own data. So, let's put your second question on hold until then.

**Comment by hsiaochiao** [comment]
Thank you so much [adRn-s](https://github.com/adRn-s) for indicating the link to the test data.
I figure out a way to solve the first problem by installing the newest version of python package (v3.10).
Thank you again
